src/System.Net.Http/src/System/Net/Http/HttpClientHandler.cs

```diff
@@ -13,6 +13,7 @@ public partial class HttpClientHandler : HttpMessageHandler
     {
         // This partial implementation contains members common to all HttpClientHandler implementations.
         private const string ManagedHandlerSettingName = "COMPlus_UseManagedHttpClientHandler";
+        private const string AppCtxManagedHandlerSettingName = "Switch.System.Net.Http.DefaultHttpClientHandlerToManagedHandler";
```

Comment:
UseManagedHttpClientHandler??

Reply:
Yeah, DefaultHttpClientHandlerToManagedHandler is pretty verbose.  It's also a little strange to talk about changing the "default", since "default" generally means it can be changed in code, but this can't.  Presumably the ""Switch.System.Net.Http." prefix is following a standard pattern, and then we could use something like "UseManagedHttpClientHandler".

Whatever we pick, the core part of the name match the environment variable, rather than using something completely different.

Reply:
I'd drop the `Switch` prefix.  I think the naming should follow the same pattern we use in Globalization.Invariant.

AppContext setting name: `System.Globalization.Invariant`
Environment variable name: `DOTNET_SYSTEM_GLOBALIZATION_INVARIANT` ( from code comment "Linux doesn't support environment variable names include dots")

Reply:
The Switch was modeled after  https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/mitigation-png-frames-in-icon-objects
Also I was wondering if it would be helpful that the environmental variable matches exactly the config prop.

Reply:
There are also the following runtimeconfig settings, that the `System.Globalization.Invariant` name was modeled after:

```
System.GC.Concurrent
System.GC.Server
System.GC.RetainVM
System.Threading.ThreadPool.MinThreads
System.Threading.ThreadPool.MaxThreads
```